gitops/snapshot_predicate_test.go

```diff
@@ -1,9 +1,12 @@
 /*
-Copyright 2022 Red Hat Inc.
```

Comment:
In fact, we should have copyright owner "Red Hat Inc" everywhere, not just year

Reply:
I did it while I was in there checking the other bits.